cloudfoundry-cli/parts/plugins/x-cf.py

```diff
@@ -0,0 +1,8 @@
+from snapcraft.plugins import go
+
+
+class CFPlugin(go.GoPlugin):
+
+    def _local_build(self):
+        self._run(['go', 'install', './{}/...'.format(
+            'github.com/cloudfoundry/cli/main')])
```

Comment:
@sergiusens @kyrofa: I couldn't find a better way to avoid the multiple main packages error. Do you have better ideas?